dracut/30ignition/ignition-generator

```diff
@@ -17,6 +17,9 @@ cmdline_arg() {
     echo "${value}"
 }
 
+# check *both* old and new keys for now
+platform_id=$(cmdline_arg ignition.platform.id "$(cmdline_arg coreos.oem.id)")
```

Comment:
Should we error if they provide both?

Reply:
Hmm, though I can see providing both as a valid strategy when migrating a distribution from one to the other. Though I guess we could still sanity check that they're the same if they're both provided.

Reply:
> Hmm, though I can see providing both as a valid strategy when migrating a distribution from one to the other

yeah I guess that kinda makes sense.